Tar driver panics when unable to find provided tar file

Example with debian tests running through bazel (note the relative path of the tar file provided):

```
➜  container-structure-test git:(master) ./out/structure-test -test.v -driver tar -image reproducible/intermediate_bundle_debian9_structure_test.tar /usr/local/google/home/nkubala/Development/debian-docker/bazel-out/local-fastbuild/bin/reproducible/debian9_structure_test.runfiles/debian_docker/tests/debian_9_test.yaml 
Using driver tar
=== RUN   TestAll
2017/11/08 14:49:43 Running tests for file /usr/local/google/home/nkubala/Development/debian-docker/bazel-out/local-fastbuild/bin/reproducible/debian9_structure_test.runfiles/debian_docker/tests/debian_9_test.yaml
=== RUN   TestAll/File_Content_Test:_Debian_Sources_Version
Retrieving image reproducible/intermediate_bundle_debian9_structure_test.tar from source Tar Archive
INFO[0000] Extracting image tar to obtain image file system 
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x75b6cd]

goroutine 7 [running]:
testing.tRunner.func1(0xc42045a1e0)
	/usr/lib/google-golang/src/testing/testing.go:711 +0x2d2
panic(0x7c3e60, 0xa67cd0)
	/usr/lib/google-golang/src/runtime/panic.go:491 +0x283
github.com/GoogleCloudPlatform/container-structure-test/types/v2.(*StructureTest).RunFileContentTests.func1(0xc42045a1e0)
	/usr/local/google/home/nkubala/Development/go/src/github.com/GoogleCloudPlatform/container-structure-test/types/v2/structure.go:119 +0xbd
testing.tRunner(0xc42045a1e0, 0xc42045e3e0)
	/usr/lib/google-golang/src/testing/testing.go:746 +0xd0
created by testing.(*T).Run
	/usr/lib/google-golang/src/testing/testing.go:789 +0x2de
```

This should exit gracefully with a message telling the user the tar file was not found.

--- a/types/v2/structure.go
+++ b/types/v2/structure.go
@@ -62,7 +62,7 @@ func (st *StructureTest) RunCommandTests(t *testing.T) int {
 			//each test needs to have its own instance of the driver: create that here
 			driver, err := st.NewDriver()
 			if err != nil {
-				t.Errorf(err.Error())
+				t.Fatalf(err.Error())
 			}
 			defer driver.Destroy()
 			vars := append(st.GlobalEnvVars, tt.EnvVars...)
@@ -85,7 +85,7 @@ func (st *StructureTest) RunFileExistenceTests(t *testing.T) int {
 			validateFileExistenceTest(t, tt)
 			driver, err := st.NewDriver()
 			if err != nil {
-				t.Errorf(err.Error())
+				t.Fatalf(err.Error())
 			}
 			defer driver.Destroy()
 			var info os.FileInfo
@@ -114,7 +114,7 @@ func (st *StructureTest) RunFileContentTests(t *testing.T) int {
 			validateFileContentTest(t, tt)
 			driver, err := st.NewDriver()
 			if err != nil {
-				t.Errorf(err.Error())
+				t.Fatalf(err.Error())
 			}
 			defer driver.Destroy()
 			actualContents, err := driver.ReadFile(t, tt.Path)
@@ -147,12 +147,12 @@ func (st *StructureTest) RunMetadataTests(t *testing.T) int {
 	t.Run(st.MetadataTest.LogName(), func(t *testing.T) {
 		driver, err := st.NewDriver()
 		if err != nil {
-			t.Errorf(err.Error())
+			t.Fatalf(err.Error())
 		}
 		defer driver.Destroy()
 		config, err := driver.GetConfig(t)
 		if err != nil {
-			t.Errorf(err.Error())
+			t.Fatalf(err.Error())
 		}
 		for _, pair := range st.MetadataTest.Env {
 			if config.Env[pair.Key] == "" {
@@ -219,7 +219,7 @@ func (st *StructureTest) RunLicenseTests(t *testing.T) int {
 		t.Run(tt.LogName(num), func(t *testing.T) {
 			driver, err := st.NewDriver()
 			if err != nil {
-				t.Errorf(err.Error())
+				t.Fatalf(err.Error())
 			}
 			defer driver.Destroy()
 			checkLicenses(t, tt, driver)

